src/app.js

```diff
@@ -25,11 +32,72 @@ const quoteData = [
   },
 ];
 
+// define templates
+let quoteTemplate;
+let tradeTemplate;
+let orderTemplate;
+
 $(document).ready(function() {
+  // templates
+  quoteTemplate = _.template($('#quote-template').html());
+  tradeTemplate = _.template($('#trade-template').html());
+  orderTemplate = _.template($('#order-template').html());
+
+  // create new quote list
   const quotes = new QuoteList(quoteData);
+  for (let quote of quoteData) {
+    quotes.add(new Quote(quote));
+  };
+
+  // create new quote list view
+  const quoteListView = new QuoteListView({
+    el: '#quotes-container',
+    model: quotes,
+    template: quoteTemplate,
+  });
+
+  quoteListView.render();
+
+  // run the simulator
   const simulator = new Simulator({
     quotes: quotes,
   });
 
   simulator.start();
+
+  // add options to select in order form
+  for (let quote of quoteData) {
+    console.log(quote);
+    $('.order-entry-form select[name=symbol]').append(`<option value='${quote.symbol}'>${quote.symbol}</option>`)
+  };
+
+  // create new empty order list
+  const orders = new OrderList();
+
+  // create new order list view
+  const orderListView = new OrderListView({
+    el: '#order-workspace',
+    model: orders,
+    template: orderTemplate,
+    quotes: quotes,
+  });
+
+  orderListView.render();
+
+  // create new trades view
+  const tradesView = new TradesView({
+    el: '#trades-container',
+    template: tradeTemplate,
+    tagName: 'li',
+    className: 'trade',
+    quotesView: quoteListView,
+    ordersView: orderListView,
```

Comment:
While passing the `QuoteListView` and `OrderListView` instances directly to `TradesView` is one way to avoid using an event bus, I think there's a more ideal approach.

If we had `Quote` models trigger the `add_trade` event then `TradesView` could listen directly to the `quotes` collection to hear any such events. While this means that the "model layer" is potentially more directly coupled, it does give a level of separation for the different UI components.

The reason I think this is preferable is that the different models and collections in the "model layer" are coupled pretty strongly already as a result of the problem that we're solving (orders and quotes are inherently related).

On the other hand, the UI could definitely be more separate -- imagine a UI comprised of these sections and several other panels. If the UI let you choose up to four panels to display at once, even if we chose not to display the Orders panel, we would still need to keep the `OrdersListView` instance active somewhere, because the `TradeListView` wouldn't work properly without it.